dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -236,7 +236,10 @@ public Response toolsIndexSearch(String query, MultivaluedMap<String, String> qu
                 int statusCode = e.getResponse().getStatusLine().getStatusCode();
                 LOG.error("Could not use Elasticsearch search", e);
                 if (ArrayUtils.contains(codesToResurface, statusCode)) {
-                    throw new CustomWebApplicationException(e.getMessage(), statusCode);
+                    String reasonPhrase = e.getResponse().getStatusLine().getReasonPhrase();
+                    // Provide a minimal amount of error information in the browser console as outlined by
+                    // https://ucsc-cgl.atlassian.net/browse/SEAB-2128
+                    throw new CustomWebApplicationException(reasonPhrase, statusCode);
                 } else {
                     throw new CustomWebApplicationException(e.getMessage(), HttpStatus.SC_INTERNAL_SERVER_ERROR);
```

Comment:
Presumably this will have the same problem?

Reply:
ReasonPhrase contains much less info. Usually a sentence about the error such as "Bad Request"

Reply:
Line 241/244 is not returning reason phrase.

Reply:
I was only fixing the information from a 400 return; but OK I guess the other exceptions should have limited info also so I will fix those.

Reply:
I shorted the message for the internal server error (500) for the first try block. However the 500 for the IOException  does not provide an easy way to get a more limited message, and also it seems very brief anyway, e.g.

Reply:
![Screen Shot 2021-03-11 at 1 24 57 PM](https://user-images.githubusercontent.com/3650921/110857081-4116ee00-826d-11eb-9f0a-a97162bbc39d.png)